src/App.vue

```diff
@@ -65,6 +71,24 @@ export default {
       this.updateLogo()
       this.updateOptions()
     })
+
+    // Sound default on
+    let localStorageSound = window.localStorage.getItem('sound')
+    this.sound = localStorageSound ? JSON.parse(localStorageSound) : true
+
+    this.gameoverSound = new Howl({
+      src: ['../static/sounds/gameover.mp3', '../static/sounds/gameover.ogg'],
+      rate: 1.3,
+      volume: 0.5
+    })
+    this.correctSound = new Howl({
+      src: ['../static/sounds/correct.mp3', '../static/sounds/correct.ogg'],
```

Comment:
Correct sound was the only one that I did not like. Can this be changed to something a little less disturbing "correct" sound?